copter/source/docs/traditional-helicopter-connecting-apm.rst

```diff
@@ -17,7 +17,7 @@ Before you begin connecting the system it is recommended to review the docs for
 Overview of servo, and RX connection
 ====================================
 
-The RC input for many ardupilot compatible autopilots is either PPM SUM (8 channels) or S.Bus (up to 18 channels).  Some controllers also accept Spektrum satellite receivers.  For receivers that only output PWM, a PPM encoder is required to connect to the autopilot.  For all helicopter frames,the RC radio requires at least 6 channels.  Traditional helicopter and quadheli frames require the flight controller have at least 5 channels. The dual helicopter frame requires the flight controller have at least 7 channels. 
+The RC input for many ardupilot compatible autopilots is either PPM SUM (8 channels) or S.Bus (up to 18 channels).  Some controllers also accept Spektrum satellite receivers.  For receivers that only output PWM, a PPM encoder is required to connect to the autopilot however this is not recommended.  For all helicopter frames, an RC radio that has at least 6 channels is required.  For flight controllers, traditional helicopter and quadheli frames require one that has at least 5 output channels. The dual helicopter frame requires the flight controller have at least 7 output channels. 
```

Comment:
Bill, why did you add this? as far as I know, using a PWM RC RX with an encoder is a valid connection method.....other than that I think its ready to merge...

Reply:
@Hwurzburg the PPM encoder is a valid connection but I have seen times where it doesn’t keep up and you see the aircraft not respond to your input. I don’t have solid proof but I don’t recommend them as a a connection method. I know for Futaba, at least, you can buy a really inexpensive receiver ($70USD) that has SBus output that can be used instead of the PPM encoder.

Reply:
yes, Sbus RX for Frsky are $11....but I know of no technical reason why a PPM encoder should be dis-recommended....its maximum delay of one frame (20ms) is unnoticeable..early days of PPM had a few units out there that did not handle all frame widths correctly....but that is a case of defective hardware design early on...we currently  recommend the encoder for other vehicles for those poor souls with old equipment....if you have other than anecodotal evidence, then we should investigate and dis-recommend for all vehicles...

Reply:
Henry, PPM encoders are ok for multicopters. But not for helicopters, especially in acro. Pilots demand more crisp control response for helicopters, which PPM encoders don't deliver. We have never recommended them for heli's.

Reply:
> but I know of no technical reason why a PPM encoder should be dis-recommended....its maximum delay of one frame (20ms) is unnoticeable.

I is very noticeable with helicopters in acro. You combine the latency of an older pwm output receiver with a PPM encoder and the helicopter handles like it's "loose" with all the linkages and swashplate bearing worn out. They are fine for all other vehicle types. It is just the nature of helicopters. Even servos used on airplanes cannot be used on helicopter cyclic.